test/grpc/account_test.dart

```diff
@@ -41,5 +41,9 @@ void main() {
       expect(users, isA<api.Users>());
       expect(users.users, hasLength(1));
     });
+
+    test('updating my account', () async {
+      await client.updateAccount(session: session, displayName: 'name');
+    });
```

Comment:
I'd expect to test a getAccount right after this, but seems like Nakama server updates account async so it's updated with a delay.